test/CMakeLists.txt

```diff
@@ -57,6 +57,13 @@ add_custom_target(run-onedpl-tests
     DEPENDS build-onedpl-tests
     COMMENT "Build and run all oneDPL tests")
 
+# Disable iterators check under Intel LLVM-based compilers on Windows
+# For details please see https://learn.microsoft.com/en-us/cpp/standard-library/iterator-debug-level?view=msvc-170
+if (WIN32 AND INTEL_LLVM_COMPILER_GNU_LIKE)
```

Comment:
Why limit this fix to just the GNU-like compilers? I just tested this on Windows with `dpcpp-cl.exe`, which is an MSVC-like compiler, and I hit the same error even with these changes.

Can we change the condition to be just `if (WIN32)`?

Reply:
Good point, will fix. Thanks!

Reply:
Fixed.